deploy-manage/upgrade/deployment-or-cluster/upgrade-717.md

```diff
@@ -1,7 +1,376 @@
+
+1. **7.17.x → 8.19.x**  
+    This brings the cluster onto the latest supported 8.x release, which is the required intermediate version before upgrading to {{version.stack}}.
+    Before running this upgrade, all ingest components and client libraries must be upgraded to 7.17.x.
+
+2. **8.19.x → {{version.stack}}**  
+    This completes the upgrade to the latest 9.x release.
+    Before running this upgrade, all ingest components and client libraries must be upgraded to 8.19.x.
+
+The following sections describe these phases in detail and point to the relevant documentation for each deployment type.
+
+Refer to [upgrade paths](/deploy-manage/upgrade.md#upgrade-paths) for more information.
+
+::::{dropdown} Alternative method: Reindex to upgrade
+For basic use cases that do not rely on {{kib}} dashboards or {{stack}} features such as {{ml}}, transforms, {{kib}} alerting, or detection rules, instead of performing two sequential upgrades (7.17 → 8.19 → {{version.stack}}), you can create a new {{version.stack}} cluster or deployment and migrate your data from the 7.17 cluster by reindexing.
+
+For detailed guidance on how to plan and execute this method, refer to [Reindex to upgrade](/deploy-manage/upgrade/prepare-to-upgrade.md#reindex-to-upgrade).
+
+It may be suitable when:
+- You prefer to build new infrastructure rather than modify an existing one.  
+- You want to reduce the risk of performing two consecutive major upgrades.  
+- You plan to redesign your topology or move to a new environment (for example, from self-managed to {{ech}} or {{ece}}).  
+
+This approach is intended for {{es}} use cases focused on indexing and querying your own data. If you need a smooth transition that preserves {{kib}} configurations and {{stack}} features data, follow the standard upgrade path instead.
+::::
+
+## Upgrade planning [planning]
+
+The [planning phase](/deploy-manage/upgrade/plan-upgrade.md) ensures that the upgrade is well understood, can be executed with minimal risk, and follows the [correct order](/deploy-manage/upgrade/plan-upgrade.md#upgrade-order).
+
+It involves defining a clear sequence of actions and assessing the impact on [service availability](/deploy-manage/upgrade.md#availability-during-upgrades) and performance during the upgrade.
+
+For the 7.17.x → 9.x upgrade path, the main planning outcome is a set of required actions to ensure compatibility across the ecosystem:
+
+* **Ingest components:**
+
+  Before the initial upgrade to 8.19.x, ensure that all ingest components ({{beats}}, {{agent}}, {{ls}}, APM) are on version 7.17.x.
+
+  After upgrading the cluster to 8.19.x and before proceeding to {{version.stack}}, upgrade all ingest components to 8.19.x.
+
+  Refer to [Upgrade your ingest components](/deploy-manage/upgrade/ingest-components.md) for detailed upgrade instructions.
```

Comment:
we'll need to link to 7.17, 8.19, and 9.x instructions here

Reply:
I'd defer the ingest components details to a next PR, so we can publish a robust version soon and then add these details for specific Logstash, Elastic Agent or Beats upgrades.